src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -15,9 +13,6 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     /// </summary>
     public class MutableObjectModelBinder : IModelBinder
     {
-        private static readonly MethodInfo CallPropertyAddRangeOpenGenericMethod =
-            typeof(MutableObjectModelBinder).GetTypeInfo().GetDeclaredMethod(nameof(CallPropertyAddRange));
```

Comment:
Yeah, I'm super jacked about this change. Always hated we had to duplicate that code.